zerver/lib/zcommand.py

```diff
@@ -32,4 +36,22 @@ def process_zcommands(content: str, user_profile: UserProfile) -> Dict[str, Any]
         ret = dict(msg=msg)
         return ret
 
+    if command == 'subscribe':
+        linked_stream = possible_linked_stream_names(content)
```

Comment:
I think we can probably do significantly better with a custom regular expression for doing this parsing precisely; the format should always be:

`^subscribe #**[^*]+** @**[^*]+**`, right?